crowbar_framework/app/models/node_object.rb

```diff
@@ -1138,6 +1138,22 @@ def ssh_cmd(cmd)
     end
   end
 
+  def net_rpc_cmd(cmd)
+    case cmd
+    when :power_cycle
+      unless system("net", "rpc", "shutdown", "-f", "-r", "-I", @node.name ,"-U", "Administrator%#{@node[:provisioner][:windows][:admin_password]}")
```

Comment:
Ok, so @ovisan have to add the package to https://github.com/crowbar/barclamp-crowbar/blob/master/chef/cookbooks/crowbar/recipes/default.rb#L29-L81 but just in case if there is any windows nodewithin crowbar available. @vuntz do you have a hint for @ovisan how he can check if we have any windows node? :)